comm.c

```diff
@@ -59,6 +59,7 @@ ssize_t read_comm_request(char **buf_ptr) {
 	if (n <= 0) {
 		return n;
 	}
+	assert(size > 0);
```

Comment:
This assert will never run because of the if statement right above. It will short-circuit there.

Reply:
Never mind, I'm too tired. It's a different variable.